specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-10-01-preview/cancelEligibility.json

```diff
@@ -0,0 +1,180 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cancel Eligibility API",
+    "version": "2021-10-01-preview",
+    "description": "Cancel Eligibility API to check whether the subscription is eligible to cancel or not."
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "CancelEligibilityModel"
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/cancelEligibility": {
+      "post": {
+        "tags": [
+          "CancelEligibilityModel"
+        ],
+        "operationId": "CancelEligibilityModel_CancelEligibility",
+        "x-ms-examples": {
+          "cancelEligibilitySubscription": {
+            "$ref": "./examples/cancelEligibilitySubscription.json"
+          }
+        },
+        "description": "The operation to check whether the subscription is eligibile to cancel.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/CancelEligibilityResponse"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CancelEligibilityResponse": {
+      "type": "object",
+      "properties": {
+        "isEligibleToCancel": {
+          "type": "boolean",
+          "description": "Is eligible to cancel"
+        }
```

Comment:
Normally we advise against booleans because they're not future proof if you want to add more values. However in this particular API I feel like it makes sense. Either you CAN cancel, or you CAN'T. There's never really supposed to be a 3rd option right.